arches/app/utils/storage_filename_generator.py

```diff
@@ -0,0 +1,5 @@
+import os
+from arches.app.models.system_settings import settings
+
+def generate_filename(instance, filename):
+    return os.sep.join([settings.UPLOADED_FILES_DIR, filename])
```

Comment:
Why use os.sep.join here?  os.path.join is a little smarter when doing these path joins.

Reply:
@aarongundel - nice catch. Made the requested change and did a cherry pick on a clean dev/7.5.x to clean up the branch.